python/lsst/ts/phosim/CloseLoopTask.py

```diff
@@ -931,13 +927,14 @@ def runWfsWep(self, obsId, butlerRootPath, instName, numPro=1, pipelineFile=None
             runProgram(
                 f"butler write-curated-calibrations {butlerRootPath} lsst.obs.lsst.Lsst{butlerInstName}"
             )
-
+        # As defined in phosim_repackager
```

Comment:
Can you just add a little more information here on what exactly `seqNum` is? I'm not clear on that still.